lib/Froxlor/Settings/Store.php

```diff
@@ -290,6 +290,28 @@ public static function storeSettingIpAddress($fieldname, $fielddata, $newfieldva
 
 	public static function storeSettingMysqlAccessHost($fieldname, $fielddata, $newfieldvalue)
 	{
+        $ips = $newfieldvalue;
+	    //Convert cidr to netmask for mysql, if needed be
+        if(strpos($ips, ',') !== false) {
+            $ips = explode(',', $ips);
+        }
+        if(is_array($ips) && count($ips) > 0) {
+            $newfieldvalue = [];
+            foreach ($ips as $ip) {
+                $org_ip = $ip;
+                $ip_cidr = explode("/", $ip);
+                if (count($ip_cidr) == 2) {
+                    $ip = $ip_cidr[0];
+                    if(in_array((int)strlen((string)$ip_cidr[1]),array(1,2))) {
```

Comment:
this looks really ugly... why not just do `if (strlen($ip_cidr[1]) <= 2) {` as it can not be 0 due to the check on line 303 if there are 2 array elements

Reply:
True, I can change that.